recipes/jellyfish/build.sh

```diff
@@ -1,4 +1,6 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+set -x -e
```

Comment:
FYI - using set -x -e will help with debugging with things go wrong. Its like setting a verbose flag in bash.

Reply:
Thanks for the tip 👍